module-audio/Audio/equalizer/Equalizer.hpp

```diff
@@ -17,19 +18,19 @@ namespace audio::equalizer
         float a2;
     };
 
-    using Equalizer = std::array<QFilterCoefficients, 5>;
+    using Equalizer = std::array<QFilterCoefficients, bands>;
```

Comment:
just curious - do you know how and why we ended up with `bands=5` ? we can set up to 9 filters up to 5 times (if I'm reading the code right :) ) it has not much to do with this PR - it has everything to do with how this code is built and you just happened to touch it therefore you might know the answer :)

Reply:
as I found in the codec datasheet, we can setup (based on the regs config) up to 7 bands. Right now it is set up for 5 bands but it was newer written explicit anywhere. I've moved it to a constexpr to be able to bond test to the proper code without using a magic number

Reply:
Thanks for the elaborate description. I don't do this often but I would support leaving a comment explaining it here :) ( I know it's not your code, and you might feel weird passing explanation in possibly codec agnostic code about the codec - but it's a rare place where I think it might be nice to inform somebody about a trap :) )

Reply:
np problem, I can describe it in a comment in the code if it's so confusing ;)

Reply:
described in the comment